src/ray/raylet/scheduling/cluster_resource_data.h

```diff
@@ -85,6 +85,8 @@ class TaskRequest {
   bool IsEmpty() const;
   /// Returns human-readable string for this task request.
   std::string DebugString() const;
+
+  TaskRequest &operator+=(const TaskRequest &other);
```

Comment:
this operator is kinda weird. it doesn't really make sense to add task requests together. can we not do this?